includes/Core/Modules/Module.php

```diff
@@ -171,11 +123,17 @@ final public function __isset( $key ) {
 	 * @return mixed Value for $key, or null if not available.
 	 */
 	final public function __get( $key ) {
-		if ( ! isset( $this->info[ $key ] ) ) {
-			return null;
+		$di = $this->get_di();
+		if ( $di->has( $key ) ) {
+			return $di->get( $key );
+		}
```

Comment:
This is exactly what I was referring to above - I'd prefer if we wouldn't use magic properties for the services (or anything in the DI container fwiw).